api/package.json

```diff
@@ -32,9 +36,18 @@
     "babel-preset-env": "^1.7.0",
     "cross-var": "^1.1.0",
     "dotenv-cli": "^7.2.1",
+    "eslint-plugin-prettier": "^5.0.0",
     "jest": "^27.2.4",
     "nock": "^13.2.0",
     "superagent": "^6.1.0",
     "supertest": "^6.1.6"
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx, jsx}": [
```

Comment:
I don't think we should be including `tsx` or `jsx` extensions here, as those are specifically for React